products/views.py

```diff
@@ -2,16 +2,106 @@
 Contains Views for products app
 """
 # pylint: disable=no-self-use, no-member
+from django.shortcuts import get_object_or_404, render
 from rest_framework import status, viewsets
-from rest_framework.authentication import TokenAuthentication
+from rest_framework.authentication import SessionAuthentication, TokenAuthentication
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
+from rest_framework.views import APIView
 
+from users.contants import CONTENT_MANAGER, ROLES, SYSTEM_ADMIN
+
+from .constants import ADD_PRODUCT, HOMEPAGE, PRODUCT_PAGE
 from .models import Product
 from .permissions import IsContentManager
 from .serializers import ProductSerializer
 
 
+class ProductHomePageView(APIView):
+    """
+    Renders the homepage of products
+    """
+
+    authentication_classes = [SessionAuthentication]
```

Comment:
Can anonymous user see the products list?

Reply:
Yes